bring-it-up/bring-it-up
# Fe #18 0 link service card component to fullpage service

## Description
#### Summary:
Generates a different path for all the different services. Leads to a services page that simply lists the service

#### Changes:
- In Services.tsx, the following code generates paths for each service within the actual API request
``` TS
<p>These are the services in database: {GetData().map(
                x => <p>{x} <Link to={generatePath("services/:id", {id:x})}>Click here</Link> </p>
                )}</p>
```
- TestService.tsx is simply a file that displays which service is opened and this allows us to see which service we'd like to test. This page renders for specific services.

#### How to test this PR:
- Start the server and connect to the MongoDB database
- Start the client side and go to the services page
- You should see every service listed out. You can feel free to click on all of them from there


## Checklist

- [ x] I performed a self-review of my code
- [x ] I ran all the tests and they were successful
